src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -24,6 +24,7 @@
     <ItemGroup>
       <MauiAsset Include="%(_MauiStaticWebAssetWithTargetPath.Identity)">
         <Link>%(_MauiStaticWebAssetWithTargetPath.TargetPath)</Link>
+        <LogicalName>%(_MauiStaticWebAssetWithTargetPath.TargetPath)</LogicalName>
```

Comment:
Can you provide info on why this is added? In theory nothing should change here. I believe @javiercn carefully crafted this MSBuild in the fires of... somewhere in Spain, I think.